### ISSUE TITLE
Segmentation Labels Map

### ISSUE THREAD

**Issue by YongchengYAO** [YzzLiu/FracSegNet#11]
Hi, 

Could you share the complete labels map for each label from 1 to 30?

reference:
Label assignment: 0 = background, 1-10 = sacrum fragment, 11-20 = left hipbone fragment, 21-30 = right hipbone fragment.

Best

**Comment by 123Ray321** [comment]
Based on my observations, the detailed labeling rules are as follows: 0 denotes the background; 1 denotes the major fragment in SA; 2–10 denote the minor fragments in SA. 11 denotes the major fragment in LI; 12–20 denote the minor fragments in LI. 21 denotes the major fragment in RI; 22–30 denote the minor fragments in RI.

**Comment by MarkXuJQ** [comment]
“This repository contains the training set of 100 CT scans with pelvic fractures and their ground-truth segmentation labels. The images and labels are stored in mha format. Each bone anatomy (sacrum, left hipbone, right hipbone) has up to 10 fragments. Bone that does not present any fracuture has only one fragment, which is itself. Label assignment: 0 = background, 1-10 = sacrum fragment, 11-20 = left hipbone fragment, 21-30 = right hipbone fragment. ”
It is on the page https://zenodo.org/records/10927452
